### PR TITLE
Getting started tutorial

### PR DESCRIPTION
**Pull request by trey-stafford**
## Description

Add tutorial for getting started with QGreenland

## Checklist

If an item on this list is done _or not needed_, simply check it with `[x]`.

- [x] Config lockfile updated (`inv config.export > qgreenland/config/cfg-lock.json`)
- [x] Version bumped if needed (`bumpversion (major|minor|patch|prerelease|build`)
- [x] CHANGELOG.md updated
- [x] Documentation updated if needed
- [x] New unit tests if needed

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -44,6 +44,15 @@
 
     discussion/*
 
+.. toctree::
```
Comment (MattF-NSIDC) [inline doc/index.rst:47]: I think tutorials should be before the how-tos. i.e. the first category.

File: doc/tutorials/get-started.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+Manual](https://docs.qgis.org/3.16/en/docs/user_manual/index.html) provides a
+detailed and comprehensive overview of QGIS' core features. Users who are new to
+Geographic Information Systems may also benefit from reviewing QGIS's [Gentle
+Introduction to
+GIS](https://docs.qgis.org/3.16/en/docs/gentle_gis_introduction/index.html)
+```
+
+## 4) Browse data in QGreenland
+
+Layers in QGreenland are organized into layer groups by category. For example,
```
Comment (MattF-NSIDC) [inline doc/tutorials/get-started.md:66]: >Layers in QGreenland are organized into layer groups

Maybe just "...organized into groups"?

File: doc/tutorials/get-started.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+The scroll wheel on your mouse or track pad can also be used to zoom in and out in the **Map view**
+```
+
+Hovering over the other map navigation tool icons will provide a tool tip
+indicating what the tool is used for. Try hovering over some of the other icons
+in the **Map Navigation Toolbar** to read their tooltips. Try out some of these
+other tools to explore the map and QGreenland's various data layers. See the
+[QGIS documentation on zooming and
+panning](https://docs.qgis.org/3.16/en/docs/user_manual/introduction/general_tools.html#zooming-and-panning)
+for more detailed information on how to effectively navigate the **Map View**.
```
Comment (MattF-NSIDC) [inline doc/tutorials/get-started.md:119]: Let's add a Summary sub-heading and review what the user learned at the end. The final bullet in this section of the Diataxis docs specifically advises against "You have learned" language, though.

https://diataxis.fr/tutorials/#the-language-of-tutorials

**Review by MattF-NSIDC — CHANGES_REQUESTED** [review]

File: doc/index.rst (hunk 1 above)
Comment (trey-stafford) [reply doc/index.rst]: f9c97cb5

File: doc/tutorials/get-started.md (hunk 2 above)
Comment (trey-stafford) [reply doc/tutorials/get-started.md]: 40a6b9bb

File: doc/tutorials/get-started.md (hunk 3 above)
Comment (trey-stafford) [reply doc/tutorials/get-started.md:120]: f6927b87

**Review by MattF-NSIDC — APPROVED** [review]

**Pull request merged**